src/Maker/MakeRegistrationForm.php

```diff
@@ -197,7 +197,7 @@ private function interactAuthenticatorQuestions(InputInterface $input, ConsoleSt
         $this->firewallName = $firewallName;
 
         // get list of guard authenticators
-        $authenticatorClasses = $interactiveSecurityHelper->getAuthenticatorClasses($firewallsData[$firewallName]);
+        $authenticatorClasses = $interactiveSecurityHelper->getAuthenticatorClasses($firewallsData[$firewallName], $securityData['security']['enable_authenticator_manager'] ?? false);
```

Comment:
The new argument might be better as a constructor argument to the class - it's kind of a "mode" that the helper should know about.